src/CarbonAware.Tools/CarbonAware.Tools.WattTimeClient/test/WattTimeClientTests.cs

```diff
@@ -522,7 +522,7 @@ public async Task GetHistoricalDataAsync_RefreshesTokenWhenNoneSet()
     }
 
     [Test]
-    public void TestClient_With_Proxy_Failure()
+    public void ClientWithProxy_ProxyFailure_ThrowsException()
```

Comment:
I wasn't sure what specifically in this test causes the proxy failure, do you to remember @juzuluag

Reply:
Hi @JenMadiedo , the test goal was to exercise the path using a non existing proxy server, so the request would failed eventually since that proxy doesn't exist.